packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -135,6 +135,19 @@ const options = {
                 retry: false
             }
         }
+    },
+    beforeHydrate: (data) => {
+        // Helper for removing the data timestamp.
+        const updateQueryTimeStamp = ({state}) => {
+            delete state.dataUpdatedAt
```

Comment:
Deleting `dataUpdatedAt` actually breaks some logic within React Query. And the end result is we're still refetching productSearch data. 

Yes, there is no skeleton shown in the UI, but the network still shows a productSearch call made client-side.

Product List page would [show a skeleton when `isRefetching`](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/ad46457ae0d1dc1e833988a84ab4217b1d7429a2/packages/template-retail-react-app/app/pages/product-list/index.jsx#L541). But what's observed now is:
1. There's already data in react-query's cache
2. react-query decides to fetch it again for some reason
3. and this refetch is not reflected in `isRefetching`
4. thus, our UI does _not_ show the skeleton

This is why I think deleting `dataUpdatedAt` breaks some internal logic in react-query.

Setting it to `Date.now()`, however, does what we wanted. The end result is:
- No skeleton shown in the UI
- And no data refetching

Reply:
Thanks. I've actually reverted to using Date.now() outlined in [this](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1912#discussion_r1688529481) comment